### ISSUE TITLE
Use plugin utPLSQL into PL/SQL Developer 14

### ISSUE THREAD

**Issue by nedim51** [utPLSQL/utPLSQL-PLSQL-Developer#30]
Hello, i use your plugin for my work and find it very useful.  I need to upgrade and add feature which lets search by tags. 
And i added necessary function into [utPLSQL-PLSQL-Developer](https://github.com/utPLSQL/utPLSQL-PLSQL-Developer) and here [utPLSQL-dotnet-api](https://github.com/utPLSQL/utPLSQL-dotnet-api) then i compiled the program with dependencies that were pointed in repository.
![image](https://github.com/utPLSQL/utPLSQL-PLSQL-Developer/assets/45014465/8f04ede9-e00a-45f1-b3f5-e583090e8d6d)
(project .NET version 4.8)

While compiling there were no errors. I use PL/SQL Developer 14 x64 and build plugin for x64 machine. After build i put in plugin into folder "PLSQL Developer 14\PlugIns\". After launching PL/SQL Developer doesn't recognize plugin in "Plugins". 

Can you tell what kind of version of libraries you used while compiling?

P.S. During compilation of release version of basic code developer also doesn't recognize plugin.

Thanks :)

**Comment by simasch** [comment]
Can you please check the log files or the event viewer to see if there are errors?
And if yes post them here?

**Comment by nedim51** [comment]
Do I understand correctly that I first need to enable debug mode and then I will see the log file in “User\AppData\Roaming\PLSQL Developer 14\Debug.txt”?

![image](https://github.com/utPLSQL/utPLSQL-PLSQL-Developer/assets/45014465/d909ab69-b389-4bbd-a07d-7c3c7fb5bf84)

**Comment by simasch** [comment]
Yes. I would also say that there should be something in the Windows Event Viewer und Application

**Comment by nedim51** [comment]
I did it in PL/SQL Developer 
[Debug.txt](https://github.com/utPLSQL/utPLSQL-PLSQL-Developer/files/15400400/Debug.txt)

![image](https://github.com/utPLSQL/utPLSQL-PLSQL-Developer/assets/45014465/b20597ee-5973-4210-b7e4-9721fc65bd99)

**Comment by nedim51** [comment]
> Yes. I would also say that there should be something in the Windows Event Viewer und Application

I'll look out the Windows Event Viewer the same way

**Comment by nedim51** [comment]
I looked at the events in Windows Event Viewer and could not find the PL/SQL Developer application in the list. I think the problem lies in the versions of the links when building the plugin in Microsoft Visual Studio
